Multisig/UI/Onboarding/TermsView.swift

```diff
@@ -64,7 +65,7 @@ struct TermsView: View {
                     .foregroundColor(.gnoHold)
                     .padding(.top, bulletTopPadding)
                 Text(text)
-                    .font(Font.gnoHeadline.weight(.medium))
+                    .font(.gnoBody)
```

Comment:
Why don't we use .body() modifier here ?

Reply:
because the default color is not needed